modelmapper/fetcher.py

```diff
@@ -0,0 +1,243 @@
+try:
+    import mmh3
+except ImportError:
+    class mmh3:
+        def hash(self, *args, **kwargs):
+            raise ImportError('Please install mmh3')
+
+import gzip
+import datetime
+import logging
+
+from collections import Mapping
+
+
+from modelmapper.base import Base
+from modelmapper.cleaner import Cleaner
+from modelmapper.slack import slack
+from modelmapper.misc import generator_chunker, generator_updater
+from modelmapper.s3 import S3Mixin
+
+try:
+    from sqlalchemy.dialects.postgresql import insert
+except ImportError:
+    def insert(table):
+        raise ImportError('Please install SQLAlchemy')
```

Comment:
I'm not sure this is necessary, isn't SQLAlchemy listed as a project dependency?

Reply:
@JackWink It is not installed by default. Only when `pip install modelmapper[fetcher]`. This is implemented so if you are just using ModelMapper to do operations that do not need SQLAlchemy, it is not going to throw an error.

Reply:
Should we update the message to say `please run "pip install modelmapper[fetcher]"`?

Reply:
@JackWink Good call. Will address that.